src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/service/listeners/notifiers/EventNotifier.kt

```diff
@@ -13,12 +13,12 @@ abstract class EventNotifier(
   private val objectMapper: ObjectMapper,
 ) : IEventNotifier {
 
-  private companion object {
+  companion object {
     val LOG: Logger = LoggerFactory.getLogger(this::class.java)
   }
 
   final override fun process(domainEvent: DomainEvent) {
-    LOG.debug("Entered process for ${this::class.java.name} ")
+    LOG.debug("Entered process for ${this::class.java.name} type: ${domainEvent.eventType}")
```

Comment:
I am seeing these warnings on Intellij. Not an issue as such and would need to be corrected in many places but is this something that we need to look at? It expects the values to be passed as arguments.

String template as argument to 'debug()' logging call